### DISCUSSION TITLE
A glitch when using Intel GPU

### DISCUSSION THREAD

**Discussion by MyWintersweet** [ocornut/imgui#5432] [New Users Build/Link/Run issues ONLY!]
I am using Dear ImGui with OpenGL to create an application on Windows10. There seems to be a very small black square flash on the screen. It appears randomly on my window and disappears very quickly. Especially when I use tables. I found that this also happens on the demo. I assume this happens because of something about window refresh. But I do not know how to solve this problem. Does anyone have a solution? Many thanks.

**Comment by rokups** [comment]
This does not sound familiar. If you could capture a screenshot or record a video that would be helpful to better understand what is going on.

**Reply by MyWintersweet** [reply]
Thank you so much for your reply. You may check issue #3033 for detail information.

**Comment by ocornut** [comment]
May be same as #3033 ?

**Reply by MyWintersweet** [reply]
Thank you so much for your reply. I've learned a lot from your project.
Yes. My problem is exactly same as issue #3033. I find that the glitch happens when I use intel GPU. And after I change GPU to NVDIA, it disappears. I will keep tracking issue #3033. Maybe you can close this discussion. Thank you again!
